### ISSUE TITLE
Yellow Ribbon:  Mock up moving "helpful links" below pagination

### ISSUE THREAD

**Issue by MarciMcGuireGCIO** [department-of-veterans-affairs/va.gov-team#29244]
## Description

On a **Yellow Ribbon** page of search results, the focus order currently flows from the Search button to the links below. Content order **should** flow from the Search button to the title above the search results, then the links below the search button, and then the footer links.

## Point of Contact

**VFS Point of Contact:** Josh Kim

## Acceptance Criteria

As a screen reader user, I want to navigate the page by tabbing and experience a meaningful content flow.

## Environment

* Operating System: all
* Browser: all
* Server destination: production

## Steps to Recreate

1. Enter https://www.va.gov/education/yellow-ribbon-participating-schools/?state=AL
 in browser
2. Tab through the page
3. Verify that the keyboard focus moves from the Search button to the links beneath it

### Content order
I checked with @andaleliz before documenting this bug, and validated this solution.

The content order should flow from the Search button and then to:
1. heading introducing the search results
1. through the individual search results
1. search results pagination
1. Helpful Links
1. footer

Edit: in order for this to be structured appropriately by headings, helpful links should be placed below the search results pagination 

### Recommendation (Josh's edit)
Ideally, the best fix would be to move the visual AND DOM location of "helpful links" to where it is placed in the content order above. In other words, It should be right below the search results pagination. (see screenshot below)

<img width="1338" alt="Screen Shot 2021-08-17 at 9 59 45 AM" src="https://user-images.githubusercontent.com/14154792/129739699-a0b6e848-657a-421f-a2e2-c8858f83d32f.png">

If moving the visual location of "helpful links" is not possible, the next alternative would be to use CSS to visually reposition it while keeping its location in the DOM right below the search results pagination.

If moving the DOM location of "helpful links" is not possible, I would recommend keeping this ticket open until a better solution can be found. I strongly recommend against using a positive `tabindex` to fix this.

#### Heading adjustments
<img width="1179" alt="Screen Shot 2021-08-17 at 10 38 00 AM" src="https://user-images.githubusercontent.com/14154792/129746580-29c495f9-c5d0-4509-a204-82668a2db6bf.png">

1. All search result titles (university names) should be `h3`s
2. Helpful links should be a `h2`

## WCAG or Vendor Guidance (optional)

* [W3C WCAG S.C. 1.3.2: Meaningful Sequence](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html)

**Comment by callen2563** [comment]
![Yellow Ribbon - Helpful Links](https://user-images.githubusercontent.com/68560244/132404849-b81a3541-1779-49f8-a16a-d5095d01277e.png)

**Comment by MarciMcGuireGCIO** [comment]
Looks great, @callen2563!

**Comment by callen2563** [comment]
![Yellow Ribbon - Helpful Links](https://user-images.githubusercontent.com/68560244/132405112-5baf1dda-c7f6-4929-a37a-9acc031636b8.png)

**Comment by MarciMcGuireGCIO** [comment]
@zacharymorel can you take a peek at Cassandra's mockup above and estimate how long it would take to make this change in Yellow Ribbon?

**Comment by zacharymorel** [comment]
> @zacharymorel can you take a peek at Cassandra's mockup above and estimate how long it would take to make this change in Yellow Ribbon?

This is perfect!
